infra/modules/service/variables.tf

```diff
@@ -66,3 +66,14 @@ variable "db_vars" {
   })
   default = null
 }
+
+variable "log_file_transition" {
+  default = {
+    STANDARD_IA = 30
+    GLACIER     = 60
+  }
+}
+
+variable "log_file_deletion" {
+  default = 2000
+}
```

Comment:
i lean towards hardcoding these as local variables for now to keep the module interface simple.